sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py

```diff
@@ -133,6 +134,26 @@ def test_translate_distribution(self):
     self.assertEqual(metric_update.distribution.count.lowBits,
                      distribution_update.count)
 
+  def test_translate_distribution_counter(self):
+    counter_update = DataflowDistributionCounter()
+    counter_update.add_input(1)
```

Comment:
<!--new_thread; commit:dd3ae5364c173a99c8a15f7e80960b9307993f7c; resolved:0-->
Can you add more inputs?  With just this one input, we have that min=max=sum=count=1, which may fail to catch mistakes.

Reply:
This test mainly focuses on testing whether ```DataflfowDistributionCounter``` can be fully converted to ```DistributionUpdate````. There is a unit test which mainly focuses on testing DistributionCounter itself.

Reply:
Right, what I mean is that since all the values as listed above are equal to 1, we may accidentally swap fields without this test noticing.  This could be improved by simply adding, for example,  `counter_update.add_input(2)`, `counter_update.add_input(3)`.

Reply:
Could you please explain more your idea? Current test case is designed for matching two data structure's field. I'm not very clear why we need to add more input.

Reply:
This is really not important at all--I just meant that if, for example, we accidentally write `dist_update_proto.max = to_split_int(distribution_update.min)` where we assign the wrong field, the current test will not catch it.

Reply:
Can you add the two lines as above? 

```
counter_update.add_input(2)
counter_update.add_input(3)
```

Reply:
for test purpose, I think tow inputs are enough. just curious why we need three inputs here?

Reply:
Thanks.  That's fine.